tslint.json

```diff
@@ -18,6 +18,7 @@
     "jsx-wrap-multiline": true,
     "object-literal-sort-keys": false,
     "quotemark": [true, "single", "jsx-double"],
+    "arrow-parens": [true, "ban-single-arg-parens"],
```

Comment:
Can we remove the arrow parens? I don't believe PatternFly requires them -- I'd like to follow their lint / prettier rules if possible?

Specifically, I'd like us to use the `printWidth` prop in `.prettierrc`


```
{
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "none",
  "useTabs": false,
  "printWidth": 120
}
```

Reply:
Thanks for digging into this, BTW. :)